### PR TITLE
ci: move Travis jobs to GitHub actions

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

**Comment by sohankunkerkar** [comment]
looks green on my fork: https://github.com/sohankunkerkar/ignition/actions/runs/415578460

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+jobs:
+  test-build:
+    name: Test build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
```
Comment (arithx) [inline .github/workflows/go.yml:16]: Is it possible to do a matrix of go versions with github actions?

Comment (sohankunkerkar) [reply .github/workflows/go.yml:16]: Yes, it's possible.

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+    - name: Install libblkid-dev
+      run: sudo apt-get install libblkid-dev
+    - name: Test container build
+      run: |
+        ./test
```
Comment (bgilbert) [inline .github/workflows/go.yml:28]: This line isn't about testing the container build specifically; it's just running the tests.

Comment (bgilbert) [reply .github/workflows/go.yml:28]: (I was thinking it would make sense to split testing into two build steps with appropriate names.)

**Review by bgilbert — APPROVED** [review]
LGTM.  @arithx?

**Review by arithx — APPROVED** [review]

**Pull request merged**